custom_components/tuya_local/devices/pool_thermometer.yaml

```diff
@@ -0,0 +1,36 @@
+name: Raddy PT-5 pool thermometer
+products:
+  - id: qirx90k6hg4qzpbz
+    model: EM3406B-TY
+entities:
+  - entity: sensor
+    name: indoor temperature
+    class: temperature
+    dps:
+      - id: 101
+        name: sensor
+        type: integer
+        class: measurement
+        unit: C
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: indoor humidity
```

Comment:
I suggest just remove the name and allow HA to use the built in translations for humidity class.